### ISSUE TITLE
add library on npm

### ISSUE THREAD

**Issue by alexgivi** [hummingbird-dev/hummingbird-treeview#9]
could you add package.json file and register library on npm? it could make it possible to add library to another projects as a dependency, without downloading, it would be great.

**Comment by hummingbird-dev** [comment]
Yes, I agree that it would make sense. I will do that if I can find some free time slot in the next weeks.

**Comment by PhyberApex** [comment]
Any update on this?

~Cheers

**Comment by hummingbird-dev** [comment]
Oh, how time flies. I put it again on my list.

**Comment by PhyberApex** [comment]
Thanks a lot! Would appreciate that! We are currently in the process of disallowing all dependencies not available on NPM...and this is kind of a blocker. Thanks for the quick reply!

~Cheers

**Comment by hummingbird-dev** [comment]
hummingbird-treeview is now available via npm. Let me know if it works for you.
